examples/macos/CommandLineSwift/Sources/main.swift

```diff
@@ -17,5 +17,5 @@ import Foundation
 // Since the Info.plist file gets embedded in the binary, we can access values
 // like the bundle identifier using the NSBundle APIs.
 let bundle = Bundle.main
-NSLog("Hello World from \(bundle.bundleIdentifier ?? "<none>")")
-NSLog("\nHere is the entire Info.plist dictionary: \(bundle.infoDictionary ?? [:])")
+let logger: Logger = .init(log: LogImp())
```

Comment:
Are all these test changes intentional? I can't tell what they're specifically testing above coverage but maybe it was just helpful for your testing? I'm working on unit tests in https://github.com/bazelbuild/rules_apple/pull/1381 that ideally we could expand to macOS too

Reply:
This was simply an attempt to make a test and get some coverage on logic, there is nothing special and I can remove it